src/mpid/ch4/src/ch4_comm.c

```diff
@@ -575,4 +579,9 @@ int MPID_Intercomm_exchange_map(MPIR_Comm * local_comm, int local_leader, MPIR_C
 /* Register CH4-specific hints */
 static void register_comm_hints(MPIR_Comm * comm)
 {
+    /* Non-standard hints for VCI selection */
+    MPIR_Comm_register_hint(MPIR_COMM_HINT_VCI_IDX_SENDER, "vci_idx_sender",
+                            MPIDI_set_comm_hint_sender_vci, MPIR_COMM_HINT_TYPE_INT, 0, -1);
+    MPIR_Comm_register_hint(MPIR_COMM_HINT_VCI_IDX_RECEIVER, "vci_idx_receiver",
+                            MPIDI_set_comm_hint_receiver_vci, MPIR_COMM_HINT_TYPE_INT, 0, -1);
```

Comment:
Isn't `sender_vci` and `receiver_vci` more straightforward to pronounce?

Reply:
Renamed in the new PR: https://github.com/pmodels/mpich/pull/5481